docs/bike-assessment/README.md

```diff
@@ -284,117 +321,122 @@ Here is our team outcome and standards:
+- The tasks are allocated based on our teams strengths, eagerness to take on certain tasks , and allowing team members to learn the most from each other and this project as possible.
 
-Ontop of this the development is broken down on a timeline of what needs to be done when
+- The Development  tasks will be broken down in Sprint planning to ensure we have enough time to complete the different parts of the project.
 
 ---
 
 ## <a id="14"></a>Discuss how Agile methodology is being implemented in your App.
 **[Home](#contents)**
 
-Daily standups are being used to check in with everyone each day and help resolve the teams blockers.
+- Daily standups are being used to check in with everyone each day and help resolve the teams blockers.
 
-Sprints - We are working on a one week sprint cycle. This allows us to prioritise our most important work in earlier sprints and leave the stretch goals for later sprints.
+- Sprints - We are working on a one week sprint cycle. This allows us to prioritise our most important work in earlier sprints and leave the stretch goals for later sprints.
 
-Retrospectives - After each sprint we have a short retrospective to look at what went right and what went wrong so we are able to adjust our behaviour for future sprints and ideally improve productivity.
+- Retrospectives - After each sprint we have a short retrospective to look at what went right and what went wrong so we are able to adjust our behaviour for future sprints and ideally improve productivity.
 
-Trello  is being used as our Kanban board to pass tickets back and forth through various points of the development cycle. A team member is assigned a ticket and it is their responsibility to see it through to completion (with the assistance of other team members as required.)
+- Trello  is being used as our Kanban board to pass tickets back and forth through various points of the development cycle. A team member is assigned a ticket and it is their responsibility to see it through to completion (with the assistance of other team members as required.)
 
 ---
 
-    ## Provide an overview and description of your Source control process.
+## <a id="141"></a>Provide an overview and description of your Source control process.
+**[Home](#contents)**
+
+For source control we will be using git through github.com. 
+
+As we are coming on to an existing project, we are working off a development branch that has been split from the the main project. In the development branch , each of the members of  our team will create their own branch in which to develop the piece of work they have been assigned. After completion and review of the team members work, they will then push their branch up into github. Once in github the code will be reviewed by the project managers and merged into the development branch via an accepted pull request. 
+
+Each new day we will 'git pull' off the master "development" branch so that we are always working off an up to date code base. 
```

Comment:
I'm being picky here, I understand your intention, but the words should read more like:
> Each time we create a new branch, we will do it from the the `develop` branch, and every day we will do a `git merge origin/develop` to make sure that the branch is as close to the develop branch as possible, resolving any conflicts earlier rather than later